### ISSUE TITLE
Instructions on how to install Optifine for Minecraft

### ISSUE THREAD

**Issue by kolgza** [system76/docs#527]
The support article instructs the user to install the Minecraft Flatpak from Flathub, instead of installing any executables from Mojang. Since third party plugins, such as optifine, assume users do the latter, additional step-by-step instructions should be added for helping users install Optifine.

**Comment by ahoneybun** [comment]
@kylerchin what are your thoughts?

**Comment by kylerchin** [comment]
Yes! That would be good! Maybe including paths for forge or fabric mods?

**Comment by kylerchin** [comment]
sorry for the late response

**Comment by ahoneybun** [comment]
No worries! Is that something that you could do?

**Comment by kylerchin** [comment]
Yes I can totally implement that

**Comment by kylerchin** [comment]
@kolgza How should people be running the game? Optifine via fabric? Or just optifine by itself?

**Comment by kylerchin** [comment]
PR #578

**Comment by kolgza** [comment]
> @kolgza How should people be running the game? Optifine via fabric? Or just optifine by itself?

I don't really know much about fabric, I've just installed Optifine on its own.

**Comment by kolgza** [comment]
Since I use the Flatpak version of Minecraft, the only thing I really needed to figure was which directory to install Optifine to.
